tests/node-project/modules/node-module/module.json

```diff
@@ -0,0 +1,12 @@
+{

```

Comment:
Update: https://github.com/Azure/dotnet-template-azure-iot-edge-module/blob/master/content/dotnet-template-azure-iot-edge-module/CSharp/module.json

Reply:
the link is not working. I see 404 page